src/coreclr/jit/lowerarmarch.cpp

```diff
@@ -2433,11 +2444,12 @@ GenTree* Lowering::TryLowerAndOrToCCMP(GenTreeOp* tree)
     tree->SetOper(GT_SETCC);
     tree->AsCC()->gtCondition = cond2;
 
-    JITDUMP("Conversion was legal. Result:\n");
-    DISPTREERANGE(BlockRange(), tree);
-    JITDUMP("\n");
+    JITDUMP("Conversion was legal. Result:\n")
+    DISPTREERANGE(BlockRange(), tree)
+    JITDUMP("\n")
```

Comment:
Why remove the semicolons? It looks odd to have statements without semicolons, even if the macro expansion allows it.

Reply:
ok will return it back, it's just that my Resharper++ highlights them as redundant which is annoying (will change its settings 🙂)

![image](https://github.com/dotnet/runtime/assets/523221/57e95a7a-dd4e-4b62-9b16-3c015379b645)